### PR TITLE
Feature/social view

### PR DESCRIPTION
**Pull request by WebStra**
Adding following features:
  - added auth poppup for login/register with errors (ajax requests);
  - added ratings for vendors;
  - writed special js-code for ajax, etc.;

### CODE REVIEW

File: resources/views/auth/login_form_modal.blade.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<div class=" display-table authentification">
+    <div class="td">
+        <h1>INTRA ÎN CONTUL TĂU</h1>
+     
+        <p class=" center">Completează formularul pentru a te loga în cont.</p>
+        <form action="{{ route('post_login') }}" class="form styled3" method="post">
+              @include('partials.errors.list')
+                <div class="col s12">
+                    <div class="input-field">
+                        <input type="email" name="email" placeholder="Email" value="{{ old('email') }}">
```
Comment (wickd) [inline resources/views/auth/login_form_modal.blade.php]: add required attribute

File: resources/views/auth/login_form_modal.blade.php [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+                    </div>
+                </div>
+                <div class="col s12">
+                    <div class="input-field">
+                        <input type="password" name="password" placeholder="Password">
```
Comment (wickd) [inline resources/views/auth/login_form_modal.blade.php]: add required attribute

File: resources/views/partials/about-seller.blade.php [hunk 3]
```diff
@@ -35,24 +43,47 @@ class="btn_ btn_base waves-effect waves-light f_small left full_width">Vezi
...
+                        { 
+                            $('.like').removeClass('vote_active');
+                            $('.unlike').addClass('vote_active');
+                        }
+                        if(out.was_liked == 'not_liked')
+                        {
+                            $('.like').removeClass('vote_active');
+                            $('.unlike').removeClass('vote_active');
+                        }
+                        ajax_running = false;
```
Comment (wickd) [inline resources/views/partials/about-seller.blade.php]: must be replaced to 
```
$.ajax({
    complete: function (){
        ajax_running = false;
    }
});
```

File: resources/views/vendors/partials/item-block-second.blade.php [hunk 4]
```diff
@@ -10,14 +10,14 @@
...
-                    <ul class="star_rating" data-rating_value="4">
+                    <ul class="star_rating" data-rating_value="{{ ($item->likes()->count()) ? (round(0.05 * (($item->likes()->count() - $item->getLikes('dislike')->count()) / $item->likes()->count() * 100)))  : '1' }}">
                         <li class="icon-star"></li>
                         <li class="icon-star"></li>
                         <li class="icon-star"></li>
                         <li class="icon-star"></li>
                         <li class="icon-star"></li>
                     </ul>
-                    <p class="small">875 păreri / 99,9% positive </p>
+                    <p class="small">{{ $item->likes()->count() }} păreri / {{  ($item->likes()->count()) ? ($item->likes()->count() - $item->getLikes('dislike')->count()) / $item->likes()->count() * 100 : '0' }} % positive </p>
```
Comment (wickd) [inline resources/views/vendors/partials/item-block-second.blade.php]: Add ```LikePresenter.php```

replace to``` LikePresenter``` on method ```renderPozitiveVotes()```

File: resources/views/vendors/partials/left-sidebar.blade.php [hunk 5]
```diff
@@ -8,14 +8,16 @@
                 </div>
                 <div class="content">
                     <h4>{{ $item->present()->renderTitle()}}</h4>
-                    <ul class="star_rating" data-rating_value="4">
+                    <ul class="star_rating" data-rating_value="{{ ($item->likes()->count()) ? (round(0.05 * (($item->likes()->count() - $item->getLikes('dislike')->count()) / $item->likes()->count() * 100)))  : '1' }}">
```
Comment (wickd) [inline resources/views/vendors/partials/left-sidebar.blade.php]: Replace to ```LikeablePresentorTrait```

File: composer.json [hunk 6]
```diff
@@ -15,7 +15,8 @@
         "rtconner/laravel-tagging": "~2.1",
         "laravel/socialite": "^2.0",
         "rtconner/laravel-likeable": "^1.2",
-        "webstra/share": "dev-master"
+        "webstra/share": "^5.2"
```
Comment (nyacide) [inline composer.json]: webstra/share min.stability

**Pull request merged**